src/functionalTest/resources/responses/response-caseworker-submit-test-case.json

```diff
@@ -0,0 +1,302 @@
+{
+  "data": {
+    "caseLinks": [],
+    "caseNameHmctsInternal": "Subject kaikaqsrf",
+    "caseFlags": {
+      "details": []
```

Comment:
As above, you can remove all empty fields as you are using IGNORING_EXTRA_FIELDS